packages/botonic-plugin-flow-builder/src/index.ts

```diff
@@ -180,10 +185,21 @@ export default class BotonicPluginFlowBuilder implements Plugin {
   }
 
   hasIntent(node: IntentNode, intent: string, locale: string): boolean {
-    const result = node.content.intents.find(
+    const intentFound = node.content.intents.find(
       i => i.locale === locale && i.values.includes(intent)
     )
-    return Boolean(result)
+    if (!intentFound) return false
+    return true
+  }
+
+  hasMetConfidenceThreshold(
+    node: IntentNode,
+    predictedConfidence: number
+  ): boolean {
+    const nodeConfidence = node.content.confidence / 100
+    const metsConfidenceThreshold = predictedConfidence >= nodeConfidence
+    if (!metsConfidenceThreshold) return false
+    return true
```

Comment:
Could be simplified to:
```suggestion
    return predictedConfidence >= nodeConfidence
```

Reply:
better as Adria says